classes/components/forms/context/PKPContextStatisticsForm.php

```diff
@@ -78,5 +78,18 @@ public function __construct(string $action, array $locales, Site $site, Context
                 'value' => $context->getData('enableInstitutionUsageStats') !== null ? $context->getData('enableInstitutionUsageStats') : $site->getData('enableInstitutionUsageStats'),
             ]));
         }
+        if ($site->getData('isSushiApiPublic') !== null && $site->getData('isSushiApiPublic')) {
+            $this->addField(new FieldOptions('isSushiApiPublic', [
+                'label' => __('manager.settings.statistics.publicSushiApi'),
+                'options' => [
+                    [
+                        'value' => true,
+                        'label' => __('manager.settings.statistics.publicSushiApi.option'),
+                    ],
+                ],
+                'default' => true,
```

Comment:
The `default` is unused here because there will always be a `value`. Same with the other form below.